sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverException.java

```diff
@@ -13,59 +11,33 @@
  *
  * @see ServiceBusErrorSource
  */
-public final class ServiceBusReceiverException extends Exception {
+public final class ServiceBusReceiverException extends AmqpException {
```

Comment:
From our discussion yesterday, I thought we decided to keep this as extending from Exception in the case that there are exceptions other than AmqpException.

Reply:
extending it from AzureException as suggested by Srikanta